web/rest/client/features/provider/contact/putContact.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Update contacts
+  In order to manage contacts
+  As a client admin
+  I need to be able to update them through the API.
+
+  @createSchema
+  Scenario: Update a contacts
+    Given I add Company Authorization header
+     When I add "Content-Type" header equal to "application/json"
+      And I add "Accept" header equal to "application/json"
+      And I send a "PUT" request to "/contacts/1" with body:
+    """
+      {
+          "name": "Updated",
+          "lastname": "Contact",
+          "otherPhone": "1004"
```

Comment:
I would try to update workPhone here

Reply:
Updated test to update a Contact with the 3 phones filled.